src/main/kotlin/net/casual/arcade/border/CenterBorderState.kt

```diff
@@ -0,0 +1,17 @@
+package net.casual.arcade.border
+
+import net.minecraft.world.level.border.BorderStatus
+import org.joml.Vector2d
+
+interface CenterBorderState {
+
+
+
+    fun getCenter(): Vector2d
```

Comment:
Maybe implement this as `getCenterX` and `getCenterZ` to not force the implmentation to use `Vector2d`, just use that in the implmentation and don't expose it